tests/31sync/06state.pl

```diff
@@ -774,7 +774,7 @@
                  qw( can_sync ) ],
 
    # sending 50 messages can take a while
-   timeout => 20000,
+   timeout => 20,
```

Comment:
Has this made tests go faster or was this just an unnecessarily large number to begin with?

Reply:
Ah, I forgot about these. I think there was a confusion for the units here, they're in seconds not milliseconds (a timeout of 6 hours seems excessive).